lib/chef/mixin/homebrew_user.rb

```diff
@@ -75,7 +75,7 @@ def calculate_owner
           owner = ::File.stat(brew_path).uid
         else
           raise Chef::Exceptions::CannotDetermineHomebrewOwner,
-            'Could not find the "brew" executable anywhere on the path.'
+            'Couldn\'t find the "brew" executable anywhere on the path.'
```

Comment:
@IanMadd is there a reason for changing these to contractions? It looks like we use the non-contraction form everywhere else, with the exception of an occasional code comment and error log. Have some concern that non-English speakers might find the contractions a little less readable.

Reply:
The general docs style guide is to use contractions:

https://learn.microsoft.com/en-us/style-guide/word-choice/use-contractions
https://developers.google.com/style/contractions